app/views/cases/offender_sar/_data_requests.html.slim

```diff
@@ -27,11 +27,15 @@
               td
                 time datetime="#{data_request.date_requested}"
                   = l data_request.date_requested, format: :default
-              td
-                = data_request.cached_num_pages
               td
                 - if data_request.cached_date_received.present?
                   time datetime="#{data_request.cached_date_received}"
                     = l data_request.cached_date_received, format: :default
+              td
+                = data_request.cached_num_pages
               td
                 = link_to t('cases.data_requests.index.link'), edit_case_data_request_path(case_details, data_request), class: 'data-requests__action'
+
+          / The 'Total' row will only be displayed if more than one data request has been made.
```

Comment:
Wondering if we even need the comment? the line below seems really clear :)

Reply:
Comment now removed.